src/main/java/com/banking/login/CredentialsDaoImpl.java

```diff
@@ -13,7 +13,7 @@
 import com.banking.exceptions.CredentialsDBAccessException;
 
 /**
- * @author Dipanjan Sengupta
+ * @author Dipanjan Sengupta 
  * @purpose - HibernateTemplate for operations on credentials table
  */
 public class CredentialsDaoImpl {
```

Comment:
- hibernate template shouldn't have a getter method since it is dependency injected
- log all TransactionException after catching them and before wrapping into custom exception
- readOnly property of Tran

sactional annotation for get all credentials, get details of a single credential should be true